holoviews/element/stats.py

```diff
@@ -0,0 +1,123 @@
+import param
+import numpy as np
+
+from ..core.dimension import Dimension, process_dimensions
+from ..core.element import Element
+from ..core.util import get_param_values
+from .chart import Chart, Scatter
+
+
+class _StatisticsElement(Chart):
```

Comment:
I would still prefer to have this called ``StatisticalElement`` which matches the name used in the unit testing class.

Reply:
Sure, don't want that appearing in the top-level namespace though.

Reply:
Can't we just set ``__all__`` so if it is used it has to be explicitly imported?

Reply:
I'm now filtering abstract classes from ``hv.element``. Works fine.